src/drivers/opengles/lv_opengles_window.c

```diff
@@ -0,0 +1,438 @@
+/**
+ * @file lv_opengles_window.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "lv_opengles_window.h"
+#if LV_USE_OPENGLES
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <memory.h>
+#include <pthread.h>
+#include <unistd.h>
+#include "../../core/lv_refr.h"
+#include "../../stdlib/lv_string.h"
+#include "../../core/lv_global.h"
+#include "../../display/lv_display_private.h"
+#include "../../lv_init.h"
+
+#include <GL/glew.h>
+#include <GLFW/glfw3.h>
+
+#include "lv_opengles_driver.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+typedef struct {
+    GLFWwindow * window;
+
+    uint8_t * fb1;
+    uint8_t * fb2;
+    uint8_t * fb_act;
+    uint8_t * buf1;
+    uint8_t * buf2;
+    uint8_t * rotated_buf;
+    size_t rotated_buf_size;
+
+    uint8_t zoom;
+    uint8_t ignore_size_chg;
+} lv_opengles_window_t;
```

Comment:
`SUGGESTION`
The phase *"window"* is a bit misleading, as the struct itself contains a window member. Suggesting `gl_disp_data_t`

Reply:
I changed it to lv_glfw_window_t, this name follows the lv_sdl_window_t. Maybe we can leave it as is, or find a better name for these kinds of structure.